packages/vscode-boxel-tools/README.md

```diff
@@ -17,11 +10,66 @@ You can also build it [locally](#development).
 
 ## Usage
 
-Open a new window and in the command palette select "Boxel Tools: Attach to Boxel Workspaces".
+### Getting Started
+
+1. Install the extension from the VS Code Marketplace or [build it locally](#development).
+2. Open a new window and in the command palette (F1 or Ctrl+Shift+P), select "Boxel Tools: Attach to Boxel Workspaces".
+3. You will be prompted to log in with your Matrix credentials:
+   - Enter your username and password
+   - If you want to use a different Matrix server, change the server URL in VS Code settings
+
+### Connecting to Boxel Workspaces
+
+After logging in, the extension will automatically discover and list all your Boxel workspaces in the "Boxel" view in the Explorer sidebar. For each workspace, you can:
+
+- **Pull workspace files**: Click the download icon next to a workspace to download all files to your local machine
+- **Push local changes**: Click the upload icon to push any local changes back to the remote workspace
+- **Enable file watching**: Toggle automatic syncing of changes by clicking the "eye" icon
+
+To add the Boxel workspaces to your VSCode workspace, use the + button in the Boxel view or the command "Add Boxel Workspaces Folder to Workspace".
+
+### Working with Workspace Files
+
+Once you've downloaded workspace files:
+
+1. Navigate to the files in your Explorer view
+2. Edit files as you normally would in VS Code
+3. If file watching is enabled, changes will automatically sync to the remote workspace
+4. Otherwise, manually push changes using the upload icon in the Boxel Workspaces view
+
+### Managing Coding Skills
+
+The extension includes a Skills Manager that allows you to enable various coding assistants:
+
+1. Find the "Boxel" view in the Explorer sidebar
+2. Browse available skills grouped by source (Base, Catalog, or your custom workspaces)
+3. Check the checkbox next to skills you want to enable
+4. Enabled skills will be applied to your editing experience automatically
+
+Skills are stored in two locations:
+- Skill definitions are saved in your user folder
```

Comment:
Do you have to worry about updating them if you change a skill in your realm? If so, how do you do it?

Reply:
There's a reload, this should be made clearer (and maybe do this on load or something, or pull any changes when we enable one).